website/docs/ref/cli.md

```diff
@@ -131,9 +131,7 @@ lingui compile
 
 #### `--overwrite` {#compile-overwrite}
 
-This command compiles message catalogs in `localeDir` and outputs minified Javascript files. Each message is replaced with a function that returns the translated message when called.
-
-Also, language data (pluralizations) are written to the message catalog as well.
+This command compiles message catalogs in `localeDir` and outputs minified JavaScript files. Each message is replaced with a function that returns the translated message when called.
```

Comment:
this also looks odd, while compiling we don't create a function, just expanding ICU syntax to be able to use it without parser in a runtime